spec/granite/form/model/validations/nested_spec.rb

```diff
@@ -143,6 +143,106 @@
       it do
         expect { subject.valid? }.not_to raise_error
         expect(subject).not_to be_valid
+        expect(subject.errors.count).to eq(1)
+      end
+    end
+  end
+
+  context 'object field is invalid and has representation of that field' do
+    before do
+      stub_class(:validated_object) do
+        include Granite::Form::Model::Validations
+        include Granite::Form::Model::Attributes
+        include Granite::Form::Model::Primary
+
+        primary :id
```

Comment:
I don't think you need the `id`, do you?

Reply:
IDK, I thought it's there for a reason. I've tried to remove it and it also doesn't affect original `object field is invalid and referenced object does not include AutosaveAssociation` context. Do you think I should remove it there as well?

Reply:
Yes